check-loaded.phpt

```diff
@@ -16,5 +16,5 @@ else {
 }
 
 ?>
---EXPECTF--
+--EXPECT--
 OK
```

Comment:
shouldn't we add empty line in the end of the files? I can see warning from github that missing line.